app/javascript/turbo.js

```diff
@@ -1,46 +1,37 @@
-import state from './state'
-import renderer from './renderer'
-import { dispatch } from './events'
+import headers from './headers'
 import lifecycle from './lifecycle'
+import { dispatch } from './events'
+import { render } from './renderer'
 
 const frameSources = {}
 
-// fires before making a turbo HTTP request
-addEventListener('turbo:before-fetch-request', event => {
-  const frame = event.target.closest('turbo-frame')
-  const { fetchOptions } = event.detail
-
-  // command invoked and busy
-  if (self.TurboBoost?.Commands?.busy) {
-    let acceptHeaders = ['text/vnd.turbo-boost.html', fetchOptions.headers['Accept']]
-    acceptHeaders = acceptHeaders.filter(entry => entry && entry.trim().length > 0).join(', ')
-    fetchOptions.headers['Accept'] = acceptHeaders
-  }
-})
-
 // fires after receiving a turbo HTTP response
 addEventListener('turbo:before-fetch-response', event => {
```

Comment:
So this is the meat of how we hook into turbo. I'm glad you asked me to have a look at the code; I am learning a lot